docs/sources/static/configuration/scraping-service.md

```diff
@@ -168,3 +168,29 @@ Entries in the API not in the synced directory will be deleted.
 container with the `grafana/agentctl` image. Tanka configurations that
 utilize `grafana/agentctl` and sync a set of configurations to the API
 are planned for the future.
+
+## Debug Ring endpoint
+
+The `/debug/ring` endpoint is a useful tool to troubleshoot issues with the scraping
+service in Scraping Service Mode. It provides information about the Distributed Hash Ring
```

Comment:
The structure of this sentence, while not grammatically incorrect, is unclear.  We could make it much more active with this rework: 

``You can use the `debug/ring` endpoint to troubleshoot issues with the scraping service in Scraping Service Mode.``